apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -203,7 +203,7 @@ export class SendMessageEmail extends SendMessageType {
       const messageVariables = contentService.buildMessageVariables(command.payload, subscriber);
       const content: IEmailBlock[] = [...emailChannel.template.content] as IEmailBlock[];
       for (const block of content) {
-        block.content = contentService.replaceVariables(block.content, messageVariables);
+        block.content = contentService.replaceVariables(block.content, messageVariables).trim();
```

Comment:
Maybe worth adding a comment here about why the `.trim` is important :P  @djabarovgeorge

Reply:
Probably it worth the commit :)
Will do